AlmaLinux icon doesn't seem to work

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I noticed that a recent pull request added icons for Alma and Rocky. I am currently running oh-my-posh on AlmaLinux9 but my prompt shows a generic Linux penguin instead of the AlmaLinux icon.

<img width="377" alt="image" src="https://github.com/JanDeDobbeleer/oh-my-posh/assets/14800051/610faa75-fc7e-42e2-9b9b-c1e668a784c4">



### Theme

{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "version": 2,
  "blocks": [
    {
      "type": "prompt",
      "alignment": "left",
      "segments": [
        {
          "type": "os",
          "style": "diamond",
          "foreground": "#26C6DA",
          "background": "#3d5058",
          "leading_diamond": "\ue0b6",
          "trailing_diamond": "\ue0b0",
          "template": " {{ if .WSL }}WSL at {{ end }}{{.Icon}} \ue0b1",
          "properties": {
            "macos": "mac"
          }
        },
        {
          "type": "session",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "#26C6DA",
          "background": "#3d5058",
          "template": " {{ if .SSHSession }}\uf817 {{ end }}{{ .UserName }}@{{ .HostName }} "
        },
        {
          "type": "shell",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "#ffffff",
          "background": "#203248",
          "template": " \ufcb5 {{ .Name }} "
        },
        {
          "type": "path",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "#193549",
          "background": "#14c2dd",
          "template": " \ue5ff {{ .Path }} ",
          "properties": {
            "folder_separator_icon": " \ue0b1 ",
            "style": "full"
          }
        },
        {
          "type": "git",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "#193549",
          "background": "#a2c4e0",
          "template": " {{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }} \uf044 {{ .Working.String }}{{ end }}{{ if and (.Staging.Changed) (.Working.Changed) }} |{{ end }}{{ if .Staging.Changed }} \uf046 {{ .Staging.String }}{{ end }} ",
          "properties": {
            "fetch_stash_count": true,
            "fetch_upstream_icon": true
          }
        },
        {
          "type": "node",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "#ffffff",
          "background": "#6CA35E",
          "template": " \ue718 {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }} "
        },
        {
          "type": "root",
          "style": "powerline",
          "powerline_symbol": "\ue0b0",
          "foreground": "#193549",
          "background": "#ffff66",
          "template": " \uf0e7 "
        },
        {
          "type": "exit",
          "style": "diamond",
          "foreground": "#ffffff",
          "background": "#007800",
          "background_templates": [
            "{{ if gt .Code 0 }}#f1184c{{ end }}"
          ],
          "trailing_diamond": "\ue0b4",
          "template": " \ufc8d{{ if gt .Code 0 }}\uf00d{{ else }}\uf42e{{ end }} ",
          "properties": {
            "always_enabled": true
          }
        }
      ]
    },
    {
      "type": "prompt",
      "alignment": "left",
      "segments": [
        {
          "type": "text",
          "style": "plain",
          "foreground": "#007ACC",
          "template": "❯ "
        }
      ],
      "newline": true
    }
  ]
}

### What OS are you seeing the problem on?

Linux

### Which shell are you using?

bash

### Log output

```Shell
Version: 17.11.2

Shell: bash (5.1.8(1)-release)

Segments:

ConsoleTitle(false)                       -   0 ms -
os(true)                                  -   0 ms -  WSL at  
session(true)                             -   0 ms -  phil@PHIL-GRAM
shell(true)                               -   0 ms -  ﲵ bash
path(true)                                -   1 ms -   ~
git(false)                                -   0 ms -
node(false)                               -   0 ms -
root(false)                               -   0 ms -
exit(true)                                -   2 ms -  ﲍ
text(true)                                -   0 ms - ❯

Run duration: 28.754722ms

Cache path: /home/phil/.cache/oh-my-posh

Config path: /mnt/c/Users/phhaw/git/init/oh-my-posh/blueish-custom.omp.json

Logs:

[DEBUG] 17:33:11.023 shell.go:Getenv:397 → NO DATA
[TRACE] 17:33:11.023 shell.go:Getenv(TERM_PROGRAM) - 17.263µs
[DEBUG] 17:33:11.023 shell.go:Getenv:397 → NO DATA
[TRACE] 17:33:11.023 shell.go:Getenv(XDG_CACHE_HOME) - 2.191µs
[TRACE] 17:33:11.023 shell_unix.go:CachePath() - 10.789µs
[DEBUG] 17:33:11.023 shell.go:Getenv:397 → /mnt/c/Users/phhaw/git/init/oh-my-posh/blueish-custom.omp.json
[TRACE] 17:33:11.023 shell.go:Getenv(POSH_THEME) - 3.519µs
[DEBUG] 17:33:11.023 shell_unix.go:Platform:71 → almalinux9
[TRACE] 17:33:11.023 shell.go:resolveConfigPath() - 10.234µs
[TRACE] 17:33:11.023 shell.go:Init() - 168.196µs
[TRACE] 17:33:11.023 shell.go:Flags() - 73ns
[TRACE] 17:33:11.046 config.go:loadConfig() - 22.258282ms
[TRACE] 17:33:11.046 shell.go:Flags() - 225ns
[DEBUG] 17:33:11.046 shell.go:Getenv:397 → NO DATA
[TRACE] 17:33:11.046 shell.go:Getenv(OMP_CACHE_DISABLED) - 4.126µs
[DEBUG] 17:33:11.046 shell.go:Shell:644 → process name: bash
[TRACE] 17:33:11.046 shell.go:Shell() - 563.616µs
[DEBUG] 17:33:11.046 shell.go:Getenv:397 → 5.1.8(1)-release
[TRACE] 17:33:11.046 shell.go:Getenv(POSH_SHELL_VERSION) - 2.762µs
[DEBUG] 17:33:11.046 debug.go:PrintDebug:23 → Segment: Title
[TRACE] 17:33:11.046 shell.go:Flags() - 47ns
[DEBUG] 17:33:11.046 block.go:Debug:280 → Segment: os
[TRACE] 17:33:11.046 shell.go:GOOS() - 83ns
[DEBUG] 17:33:11.046 shell.go:Pwd:427 → /home/phil
[TRACE] 17:33:11.046 shell.go:Pwd() - 10.335µs
[TRACE] 17:33:11.046 shell.go:GOOS() - 45ns
[TRACE] 17:33:11.046 shell.go:GOOS() - 29ns
[DEBUG] 17:33:11.046 shell_unix.go:Platform:71 → almalinux9
[DEBUG] 17:33:11.046 properties.go:GetBool:22 → display_distro_name: false
[DEBUG] 17:33:11.046 properties.go:GetString:28 → NO DATA
[DEBUG] 17:33:11.046 properties.go:GetString:28 → 
[TRACE] 17:33:11.046 shell_unix.go:Root() - 313ns
[TRACE] 17:33:11.046 shell.go:Shell() - 76ns
[TRACE] 17:33:11.046 shell.go:ErrorCode() - 54ns
[DEBUG] 17:33:11.046 shell_unix.go:IsWsl:35 → true
[TRACE] 17:33:11.046 shell_unix.go:IsWsl() - 3.273µs
[TRACE] 17:33:11.046 shell.go:Pwd() - 83ns
[TRACE] 17:33:11.046 shell.go:GOOS() - 44ns
[TRACE] 17:33:11.046 shell.go:GOOS() - 27ns
[TRACE] 17:33:11.046 shell.go:GOOS() - 27ns
[TRACE] 17:33:11.046 shell.go:GOOS() - 28ns
[TRACE] 17:33:11.046 shell.go:GOOS() - 66ns
[TRACE] 17:33:11.046 shell.go:GOOS() - 27ns
[DEBUG] 17:33:11.046 shell.go:User:548 → phil
[TRACE] 17:33:11.046 shell.go:User() - 2.164µs
[DEBUG] 17:33:11.046 shell.go:Host:560 → PHIL-GRAM
[TRACE] 17:33:11.046 shell.go:Host() - 3.711µs
[TRACE] 17:33:11.046 shell.go:GOOS() - 27ns
[DEBUG] 17:33:11.046 shell_unix.go:Platform:71 → almalinux9
[DEBUG] 17:33:11.046 shell.go:Getenv:397 → 1
[TRACE] 17:33:11.046 shell.go:Getenv(SHLVL) - 1.92µs
[TRACE] 17:33:11.046 shell.go:TemplateCache() - 55.424µs
[TRACE] 17:33:11.046 shell.go:TemplateCache() - 201ns
[TRACE] 17:33:11.047 shell.go:Shell() - 78ns
[DEBUG] 17:33:11.047 block.go:Debug:280 → Segment: session
[TRACE] 17:33:11.047 shell.go:Pwd() - 74ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 45ns
[DEBUG] 17:33:11.047 shell.go:Getenv:397 → NO DATA
[TRACE] 17:33:11.047 shell.go:Getenv(SSH_CONNECTION) - 3.078µs
[DEBUG] 17:33:11.047 shell.go:Getenv:397 → NO DATA
[TRACE] 17:33:11.047 shell.go:Getenv(SSH_CLIENT) - 1.6µs
[TRACE] 17:33:11.047 shell.go:TemplateCache() - 101ns
[TRACE] 17:33:11.047 shell.go:TemplateCache() - 157ns
[TRACE] 17:33:11.047 shell.go:Shell() - 61ns
[DEBUG] 17:33:11.047 block.go:Debug:280 → Segment: shell
[TRACE] 17:33:11.047 shell.go:Pwd() - 78ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 44ns
[DEBUG] 17:33:11.047 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[TRACE] 17:33:11.047 shell.go:Shell() - 37ns
[TRACE] 17:33:11.047 shell.go:Flags() - 78ns
[TRACE] 17:33:11.047 shell.go:TemplateCache() - 63ns
[TRACE] 17:33:11.047 shell.go:TemplateCache() - 294ns
[TRACE] 17:33:11.047 shell.go:Shell() - 51ns
[DEBUG] 17:33:11.047 block.go:Debug:280 → Segment: path
[TRACE] 17:33:11.047 shell.go:Pwd() - 73ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 50ns
[TRACE] 17:33:11.047 shell.go:Pwd() - 49ns
[TRACE] 17:33:11.047 shell.go:Shell() - 93ns
[TRACE] 17:33:11.047 shell.go:Shell() - 35ns
[DEBUG] 17:33:11.047 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 17:33:11.047 properties.go:GetString:28 → 
[DEBUG] 17:33:11.047 properties.go:GetString:28 → 
[TRACE] 17:33:11.047 shell.go:GOOS() - 65ns
[DEBUG] 17:33:11.047 properties.go:GetString:28 → ~
[DEBUG] 17:33:11.047 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 17:33:11.047 shell.go:PathSeparator() - 61ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 27ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 32ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 41ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 26ns
[TRACE] 17:33:11.047 shell.go:PathSeparator() - 26ns
[TRACE] 17:33:11.047 shell.go:PathSeparator() - 26ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 26ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 27ns
[TRACE] 17:33:11.047 shell.go:PathSeparator() - 43ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 26ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 26ns
[TRACE] 17:33:11.047 shell.go:PathSeparator() - 27ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 27ns
[TRACE] 17:33:11.047 shell.go:GOOS() - 25ns
[TRACE] 17:33:11.047 shell.go:PathSeparator() - 28ns
[TRACE] 17:33:11.047 shell.go:Pwd() - 69ns
[DEBUG] 17:33:11.047 shell_unix.go:IsWsl:35 → true
[TRACE] 17:33:11.047 shell_unix.go:IsWsl() - 11.024µs
[DEBUG] 17:33:11.048 shell.go:RunCommand:578 → //wsl.localhost/AlmaLinux9/home/phil
[TRACE] 17:33:11.048 shell.go:RunCommand(wslpath -m /home/phil) - 760.733µs
[TRACE] 17:33:11.048 shell.go:StackCount() - 241ns
[TRACE] 17:33:11.048 shell_unix.go:DirIsWritable(/home/phil) - 5.039µs
[TRACE] 17:33:11.048 shell.go:TemplateCache() - 557ns
[TRACE] 17:33:11.048 shell.go:TemplateCache() - 220ns
[TRACE] 17:33:11.048 shell.go:Shell() - 132ns
[DEBUG] 17:33:11.048 block.go:Debug:280 → Segment: git
[TRACE] 17:33:11.048 shell.go:Pwd() - 181ns
[TRACE] 17:33:11.048 shell.go:GOOS() - 69ns
[DEBUG] 17:33:11.048 shell_unix.go:IsWsl:35 → true
[TRACE] 17:33:11.048 shell_unix.go:IsWsl() - 2.393µs
[DEBUG] 17:33:11.048 shell.go:FileContent:522 ↓
    5.15.90.1-microsoft-standard-WSL2

[TRACE] 17:33:11.048 shell.go:FileContent(/proc/sys/kernel/osrelease) - 44.006µs
[TRACE] 17:33:11.048 shell_unix.go:IsWsl2() - 50.809µs
[TRACE] 17:33:11.048 shell.go:Pwd() - 74ns
[DEBUG] 17:33:11.049 shell.go:RunCommand:578 → //wsl.localhost/AlmaLinux9/home/phil
[TRACE] 17:33:11.049 shell.go:RunCommand(wslpath -m /home/phil) - 702.287µs
[TRACE] 17:33:11.049 shell.go:GOOS() - 75ns
[DEBUG] 17:33:11.049 shell.go:CommandPath:599 → /usr/bin/git
[TRACE] 17:33:11.049 shell.go:CommandPath(git) - 22.677µs
[TRACE] 17:33:11.049 shell.go:HasCommand(git) - 34.085µs
[TRACE] 17:33:11.049 shell.go:Pwd() - 71ns
[ERROR] 17:33:11.049 shell.go:HasParentFilePath:732 → stat .git: no such file or directory
[TRACE] 17:33:11.049 shell.go:HasParentFilePath(.git) - 20.203µs
[DEBUG] 17:33:11.049 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 17:33:11.049 block.go:Debug:280 → Segment: node
[TRACE] 17:33:11.049 shell.go:Pwd() - 65ns
[TRACE] 17:33:11.049 shell.go:GOOS() - 49ns
[DEBUG] 17:33:11.049 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .npmrc .vue]
[DEBUG] 17:33:11.049 properties.go:GetStringArray:52 → folders: []
[DEBUG] 17:33:11.049 properties.go:GetBool:22 → home_enabled: false
[TRACE] 17:33:11.049 shell.go:Pwd() - 41ns
[DEBUG] 17:33:11.049 block.go:Debug:280 → Segment: root
[TRACE] 17:33:11.049 shell.go:Pwd() - 39ns
[TRACE] 17:33:11.049 shell.go:GOOS() - 43ns
[TRACE] 17:33:11.049 shell_unix.go:Root() - 256ns
[DEBUG] 17:33:11.049 block.go:Debug:280 → Segment: exit
[TRACE] 17:33:11.050 shell.go:Pwd() - 41ns
[TRACE] 17:33:11.050 shell.go:GOOS() - 121ns
[TRACE] 17:33:11.050 shell.go:ErrorCode() - 71ns
[DEBUG] 17:33:11.050 properties.go:GetBool:22 → always_enabled: true
[TRACE] 17:33:11.050 shell.go:TemplateCache() - 308ns
[TRACE] 17:33:11.052 shell.go:TemplateCache() - 556ns
[TRACE] 17:33:11.052 shell.go:Shell() - 149ns
[TRACE] 17:33:11.052 shell.go:TemplateCache() - 256ns
[TRACE] 17:33:11.052 shell.go:Flags() - 51ns
[DEBUG] 17:33:11.052 block.go:Debug:280 → Segment: text
[TRACE] 17:33:11.052 shell.go:Pwd() - 83ns
[TRACE] 17:33:11.052 shell.go:GOOS() - 72ns
[TRACE] 17:33:11.052 shell.go:TemplateCache() - 61ns
[TRACE] 17:33:11.052 shell.go:Shell() - 57ns
[DEBUG] 17:33:11.052 shell.go:Getenv:397 → NO DATA
[TRACE] 17:33:11.052 shell.go:Getenv(XDG_CACHE_HOME) - 3.042µs
[TRACE] 17:33:11.052 shell_unix.go:CachePath() - 18.538µs
[TRACE] 17:33:11.052 shell.go:Flags() - 92ns
```

--- a/src/platform/shell.go
+++ b/src/platform/shell.go
@@ -771,14 +771,14 @@ func (env *Shell) LoadTemplateCache() {
 	if !OK {
 		return
 	}
-	var templateCache TemplateCache
-	err := json.Unmarshal([]byte(val), &templateCache)
+	var tmplCache TemplateCache
+	err := json.Unmarshal([]byte(val), &tmplCache)
 	if err != nil {
 		env.Error(err)
 		return
 	}
-	templateCache.initialized = true
-	env.tmplCache = &templateCache
+	tmplCache.initialized = true
+	env.tmplCache = &tmplCache
 }
 
 func (env *Shell) Logs() string {
@@ -811,6 +811,7 @@ func (env *Shell) TemplateCache() *TemplateCache {
 
 	const separator = "="
 	values := os.Environ()
+	env.DebugF("environment: %v", values)
 	for value := range values {
 		key, val, valid := strings.Cut(values[value], separator)
 		if !valid {

--- a/src/segments/os.go
+++ b/src/segments/os.go
@@ -51,6 +51,7 @@ func (oi *Os) Enabled() bool {
 func (oi *Os) getDistroIcon(distro string) string {
 	iconMap := map[string]string{
 		"alma":                "\uF31D",
+		"almalinux9":          "\uF31D",
 		"alpine":              "\uF300",
 		"aosc":                "\uF301",
 		"arch":                "\uF303",

